symfony/framework-bundle/4.2/src/Kernel.php

```diff
@@ -31,7 +31,7 @@ public function registerBundles()
     {
         $contents = require $this->getProjectDir().'/config/bundles.php';
         foreach ($contents as $class => $envs) {
-            if (isset($envs['all']) || isset($envs[$this->environment])) {
+            if (($envs['all'] ?? false) && ($envs[$this->environment] ?? true) || ($envs[$this->environment] ?? false)) {
```

Comment:
For the record: this was rewritten as `if ($envs[$this->environment] ?? $envs['all'] ?? false) {` and backported to the 3.3 recipe in #491